docker-compose.yml

```diff
@@ -25,22 +28,22 @@ services:
         constraints: [node.role == manager]
   ui:
     image: dockervoyagerlabs/dfproxy:1.7.4
-    ports:
```

Comment:
There's no reason to publish the ports if the requests are forwarded through the proxy.